rust/connlib/libs/client/src/messages.rs

```diff
@@ -57,6 +58,26 @@ pub enum IngressMessages {
     ResourceAdded(ResourceDescription),
     ResourceRemoved(RemoveResource),
     ResourceUpdated(ResourceDescription),
+
+    IceCandidates(GatewayIceCandidates),
+}
+
+/// A gateway's ice candidate message.
+#[derive(Debug, Deserialize, Serialize, Clone, PartialEq, Eq)]
+pub struct BroadcastGatewayIceCandidates {
+    /// Gateway's id the ice candidates are meant for
+    pub gateway_ids: Vec<Id>,
+    /// Actual RTC ice candidates
+    pub candidates: Vec<RTCIceCandidateInit>,
```

Comment:
It might be wise to define our own data structure here. This is part of our API and needs to be backwards compatible once we deploy this to users.

A library update could change the format of `RTCIceCandidateInit` which would break the API.

Reply:
I'd rather keep this for now, since we **need** to define our own, much simpler struct, when we discard webrtc